dashboard/app/views/pd/workshop_mailer/teacher_enrollment_receipt.html.haml

```diff
@@ -19,7 +19,9 @@
     Thanks for enrolling in Code.org’s
     = @workshop.course_name
     = @workshop.subject
-    workshop!
+    - if @workshop.course != Pd::Workshop::COURSE_CSF
+      workshop
+    !
```

Comment:
I previously suggested "Intro" and "Deep Dive" to Jenna but she preferred the full names. Let me double check with Sarah.

If we do keep the longer name, I'll update to check against the course name instead of hard coding to CSF.